### PR TITLE
Update testing JWT Authentication with ApiTestCase

### PR DESCRIPTION
**Pull request by GrimpEreau**

### CODE REVIEW

File: core/jwt.md [hunk 1]
```diff
@@ -298,69 +298,59 @@ services:
...
 <?php
-// features/bootstrap/FeatureContext.php
+// tests/AuthenticationTest.php
+
+namespace App\Tests;
 
 use App\Entity\User;
-use Behat\Behat\Hook\Scope\BeforeScenarioScope;
-use Behatch\Context\RestContext;
+use ApiPlatform\Core\Bridge\Symfony\Bundle\Test\ApiTestCase;
```
Comment (alanpoulain) [inline core/jwt.md:312]: Should be before?

File: core/jwt.md [hunk 2]
```diff
@@ -298,69 +298,59 @@ services:
...
-    {
-        $user = new User();
-        $user->setUsername('admin');
-        $user->setPassword('ATestPassword');
-        $user->setEmail('test@test.com');
+    use ReloadDatabaseTrait;
 
-        $this->manager->persist($user);
-        $this->manager->flush();
+    public function testLogin(): void {
```
Comment (alanpoulain) [inline core/jwt.md:319]:
```suggestion
    public function testLogin(): void
    {
```

File: core/jwt.md [hunk 3]
```diff
@@ -298,69 +298,59 @@ services:
         autoconfigure: false
 ```
 
-## Testing with Behat
+## Testing
 
-Let's configure Behat to automatically send an `Authorization` HTTP header containing a valid JWT token when a scenario is marked with a `@login` annotation. Edit `features/bootstrap/FeatureContext.php` and add the following methods:
+To test your authentication with ApiTestCase, you can write a method as below:
```
Comment (dunglas) [inline core/jwt.md:303]:
```suggestion
To test your authentication with `ApiTestCase`, you can write a method as below:
```

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks @GrimpEreau!

**Pull request merged**